src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2103 @@
+# New Tracker
+
+Version 2.36 of DHIS2 introduced a set of brand new tracker endpoints dedicated to importing and querying tracker objects (Including tracked entities, enrollments, events, and relationships).
+These new endpoints set a discontinuity with earlier implementations. Re-engineering the endpoints allowed developers to improve, redesign, and formalize the API's behavior to improve the Tracker services.
+
+The newly introduced endpoints consist of:
+* `POST /api/tracker`
+* `GET /api/enrollments`
+* `GET /api/events`
+* `GET /api/trackedEntities`
+* `GET /api/relationships`
+
+Significant changes occurred in version 2.36 to make the interface with clients homogeneous and to allow more consistent and flexible ways to use the services.
+
+> ***NOTE***
+>
+> The old endpoints are marked as deprecated but still work as before.<br>
+> Some functionality is not yet ready in the new endpoints, but they support their primary use-cases.
+
+
+> ***NOTE 2***
+>
+> These endpoints currently only support the `JSON` format as input/output.<br>
+> Support for the `CSV` format will also be available in the future.
+
+## Changes in the API
+
+Property names used in the API have changed to use consistent naming across all the new endpoints.
+
+### Tracker Import changelog (`POST`)
+
+The following table highlights the differences between the old tracker import endpoints (version 2.35) and the new endpoint (introduced in version 2.36).
+
+|Tracker Object|V2.35|V2.36|
```

Comment:
would try to find other headings for these tables, 2.35 and 2.36 gives a bit the wrong impression?